api/config.go

```diff
@@ -56,9 +71,13 @@ Failure Threshold: {{.FailureThreshold}}
 Exec Probe: {{or .Exec "N/A"}}
 HTTP GET Probe: {{or .HTTPGet "N/A"}}
 TCP Socket Probe: {{or .TCPSocket "N/A"}}`)
-	if err != nil { panic(err) }
+	if err != nil {
+		panic(err)
+	}
 	err = tmpl.Execute(&doc, h)
```

Comment:
I think this is out of scope for doc PR. I'll fix it up in a subsequent PR. I guess my auto-formatter just caught it by default.